comment/validators.py

```diff
@@ -135,3 +140,37 @@ def is_valid(self):
             return True
         except ValidationError:
             return False
+
+
+@lru_cache()
+def _get_allowed_orders():
+    allowed_orders = ['reaction__likes', 'reaction__dislikes', 'posted']
+    # adds support for descending order as well
+    allowed_orders.extend(list(map(lambda a: '-' + a, allowed_orders)))
+    # django allows this to support random order
+    allowed_orders.append('?')
+    return allowed_orders
+
+
+def _validate_orders_are_unique(preferred_orders):
```

Comment:
Wouldn't be this name more eligible:
```suggestion
def _validate_unique_orders(preferred_orders):
```

Reply:
validate orders are unique seems more grammatically correct to me over validate unique orders. To me, the former seems to not do something that we intended.

Reply:
Are we writing a novel? 😄 In fact I usually don't see verb `to be` inside the function names.

Reply:
on reading validate unique orders, the first though that comes to my mind is this function must be doing is validating orders which are unique(i.e. already unique orders have been passed to it, and it is performing some validation over them), while what we intended to do is to validate whether orders are unique or not.

Reply:
I do agree with you but this name `_validate_unique_orders` what I usally see for this kind of function/methods. Feel free to keep your naming.

Reply:
I think `are_orders_unique` seems to be striking the right chords to me. Right?

Reply:
on second thoughts, doesn't seem to be a great choice since `are_orders_unique` should just be returning `bool` and not raising exceptions, if not.